src/parsec_security/parsec_threat_model/threat_model.md

```diff
@@ -289,15 +291,15 @@ Attacker with access to the log stream generated by the Parsec service and to th
 Attacker with access to the configuration file for the Parsec service or to the OS-provided
 mechanism for reading it.
 
-|   | Description                                                                                                                   | Mitigation    | Assets   |
-|---|-------------------------------------------------------------------------------------------------------------------------------|---------------|----------|
-| S | An adversary can spoof a configuration file, leading to the service operating in an ineffective state or not starting at all. | O-6           | AS5, AS6 |
-| T | The configuration file set up by the administrator might be modified, leading to the service being ineffective or broken.     | O-6           | AS5, AS6 |
-| R | There are no guarantees that the person setting up the configuration file was authorised to do so.                            | O-6           |          |
-| I | The configuration file might, if read by an adversary, provide information that opens different attack avenues.               | O-6           | AS5      |
-| I | If the configuration file is tampered, information about the existing keys can be extracted through rogue providers.          | O-6           | AS5      |
-| D | Removing or altering the configuration file can lead to the Parsec service not starting or working in a broken state.         | O-6           | AS6      |
-| E | Parsing a malicious configuration file can lead to code execution at the Parsec privilege level.                              | M-5, U-3, O-6 | All      |
+|   | Description                                                                                                                                   | Mitigation    | Assets        |
+|---|-----------------------------------------------------------------------------------------------------------------------------------------------|---------------|---------------|
+| S | An adversary can spoof a configuration file, leading to the service operating in an unsecure state, ineffective state or not starting at all. | O-6           | AS4, AS5, AS6 |
+| T | The configuration file set up by the administrator might be modified, leading to the service being unsecure, ineffective or broken.           | O-6           | AS4, AS5, AS6 |
```

Comment:
Hmmm, why client data? I don't think the admin role allows you to actually view any client-related info (maybe just the client names?)

Reply:
My thinking was that if the attacker can get themself to be an admin, they can call `DeleteClient` on all the clients which impacts client data.

Reply:
Hmm, I don't think it's the data that's not available, at most the keys are not available (which could be used to then read/sign/whatever the data)

Reply:
Ah right! In my head it was the same client's data as in the description of the operation. I can replace with AS3 "Private Keys" for availability then?

Reply:
Yes, that's what I was thinking!